### PR TITLE
[Swift 2.3] Fix crash for looking up sim id on real device

### PR DESCRIPTION
**Pull request by kdawgwilk**
This fixes a crash when trying to lookup a simulator identifier on a real device.  Fixes Issue #60

### CODE REVIEW

**Comment by kdawgwilk** [comment]
Hold off on this...I thought it was the force unwrap causing the crash but it appears to be the `getenv()` call that is crashing

**Comment by kdawgwilk** [comment]
Alright this is ready to go. I found a better API to get the ENV var for the simulator that won't crash if it doesn't exist. and I give a default "Unknown" value as the identifier because I tried the identifier passed in but it caused and infinite recursion :(

**Comment by dennisweissmann** [comment]
Hi, this one looks just as good as the Swift 3 one, however it doesn't build because line length is now 162 instead of 160 (this is an issue coming from our swiftlint rules. 2 possible fixes: change the rule in .swiftlint.yml or split the statement over multiple lines (I'd prefer the latter). Other than that: 💯🎉🙏 :)

File: Source/Device.swift [hunk 1]
```diff
@@ -240,15 +240,17 @@ public enum Device {
       case "iPad5,1", "iPad5,2":                      return iPadMini4
       case "iPad6,3", "iPad6,4":                      return iPadPro9Inch
       case "iPad6,7", "iPad6,8":                      return iPadPro12Inch
-      // swiftlint:disable:next force_unwrapping
-      case "i386", "x86_64":                          return Simulator(mapIdentifierToDevice(String(UTF8String: getenv("SIMULATOR_MODEL_IDENTIFIER"))!))
+      case "i386", "x86_64":                          return Simulator(mapIdentifierToDevice(
+                                                        NSProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"] ?? "iOS"
+                                                      ))
```
Comment (kdawgwilk) [inline Source/Device.swift]: It looks weird here in the diff but it should look alright in Xcode

**Comment by dennisweissmann** [comment]
Perfect, thank you so much! :)

**Pull request merged**